schema/workflow/unit/io/object_storage.json

```diff
@@ -0,0 +1,29 @@
+{
+    "schemaId": "workflow-unit-io-object-storage",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "object-storage io schema",
+    "type": "object",
+    "allof": [
+        {
+            "$ref": "../../unit.json"
+        }
+    ],
+    "properties": {
+        "filename": {
```

Comment:
Let's use the pathname and basename to be clear? If pathname is empty, we assume current directory

Reply:
Filename has been split into pathname and basename